ci/azure-wheel-test-upload.yml

```diff
@@ -24,23 +24,23 @@ steps:
   - task: TwineAuthenticate@1
     inputs:
       pythonUploadServiceConnection: AzureNeuronPypiNightly
-    condition: and(succeeded(), in(variables['Build.Reason'], 'Manual', 'Schedule'), ne(variables.GPU_BUILD, 'true'), ne(variables['NRN_NIGHTLY_UPLOAD'], 'false'))
+    condition: and(succeeded(), in(variables['Build.Reason'], 'Manual', 'Schedule'), ne(variables.GPU_BUILD, 'true'), ne(variables['NRN_NIGHTLY_UPLOAD'], 'false'), eq(variables['Build.SourceBranch'], 'refs/heads/master'))
```

Comment:
> eq(variables['Build.SourceBranch'], 'refs/heads/master')

This is OK in principle but just curious why this check is/was required. (In case of "emergencies / hacks / workarounds", this was a way to upload wheel from different branch).

By the way, here are more/detailed instructions which could be linked https://github.com/neuronsimulator/nrn/issues/877

Reply:
Better be safe than sorry. We now need to add only 2 variables (instead of 3)

> In case of "emergencies / hacks / workarounds"

We have too many of those.

> By the way, here are more/detailed instructions which could be linked #877

Ok your last comment should go in.

Reply:
Just end up here with `git log -p` to find out why we constrained nightly upload to `refs/heads/master`  :)

Reply:
heh, my "assert" triggered. Forgot about this